src/chocolatey.tests.integration/scenarios/UninstallScenarios.cs

```diff
@@ -1065,5 +1065,256 @@ public void should_have_expected_error_in_package_result()
                 errorFound.ShouldBeTrue();
             }
         }
+
+
+        public class when_uninstalling_a_hook_package : ScenariosBase
+        {
+            private PackageResult packageResult;

```

Comment:
```suggestion
            private PackageResult _packageResult;

```
Should be consistent with private variable naming.